docker/ansible-test/Dockerfile

```diff
@@ -23,6 +23,9 @@ RUN apt-get update -y \
     && chmod +x /entrypoint
 
 ENV HOME /
+ENV LOGNAME=user1
+ENV USER=user1
```

Comment:
Can you please explain the purpose of these environment variable names.
Why are you setting `USER` variable explicitly

Reply:
I was trying to resolve an issue where a python lib (ansible-doc called by ansible-test sanity) is calling `getpass.getuser()` throwing a `getpwuid(): uid not found` error since the OCP container is rootless and assigned a random user id that doesn't match any users in the pod's os. But this did not resolve it, I will keep working this issue separately.